### PR TITLE
YLP-510 Display pending invitation members

### PR DESCRIPTION
**Pull request by yvon-dblg**
Not found!

### CODE REVIEW

File: packages/yourloops/pages/hcp/team-members-table.tsx [hunk 1]
```diff
@@ -89,72 +95,108 @@ function MembersTableBody(props: TeamMembersProps): JSX.Element {
...
+        <Tooltip title={t("team-member-pending") as string} aria-label={t("team-member-pending")} placement="bottom">
+          <AccessTimeIcon />
+        </Tooltip>
       );
     }
 
-    let removeMemberButton: JSX.Element | null = null;
     if (userIsAdmin && userId !== currentUserId) {
       const handleClickRemoveMember = async (): Promise<void> => {
         await onShowRemoveTeamMemberDialog(member);
```
Comment (franck-fourel) [inline packages/yourloops/pages/hcp/team-members-table.tsx:180]: I would expect that we manage differently the remove of effective members and the remove of pending members.

**Review by franck-fourel — COMMENTED** [review]
A comment on the way we plan to manage the delete of the pending members. I don't know if this point has been discussed.

File: packages/yourloops/pages/hcp/team-members-table.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/team-members-table.tsx:180]: No, for now, nothing is different I think.
@MarineDBL An advice on this one ?

Comment (franck-fourel) [reply packages/yourloops/pages/hcp/team-members-table.tsx:181]: That's correct. Something that can be managed later on. 
Anyway we have to keep in mind that api calls will be different as we will have to disable the pending notification. I don't think there is a route defined for that in hydrophone.

**Review by Nas84 — APPROVED** [review]
LGTM

**Pull request merged**